src/components/screens/disableBioAuth/styles.js

```diff
@@ -1,31 +1,46 @@
-import { themes, colors, boxes } from 'constants/styleGuide';
+import { deviceHeight, SCREEN_HEIGHTS } from 'utilities/device';
+import { themes, colors, fonts } from 'constants/styleGuide';
+
+const isSmallScreen = deviceHeight() < SCREEN_HEIGHTS.SM;
 
 export default () => ({
   common: {
-    wrapper: {
-      flex: 1,
-      backgroundColor: colors.light.white,
+    heading: {
```

Comment:
```suggestion
    header: {
```
so we maintain the nomenclature used in other components?